faq.md

```diff
@@ -385,11 +390,10 @@ You can compile and deploy your Flutter app to iOS and Android.
 Mobile operating systems: Android Jelly Bean, v16, 4.1.x or newer, and
 iOS 8 or newer.
 
-Mobile hardware: 64-bit iOS devices (starting with iPhone 5S and newer
+Mobile hardware: iOS devices (starting with iPhone 5 and newer
```

Comment:
@cbracken can give you the data.  I think we run on the 4S just fine. :)

Reply:
Correct. iPhone 4S is our baseline and what we now test in the devicelab. Minor difference, but matters, since it's an older armv7 processor vs the armv7s available in the iPhone 5.